partner_identification/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2004-2010 Tiny SPRL http://tiny.be
+# Copyright 2010-2012 ChriCar Beteiligungs- und Beratungs- GmbH
+#             http://www.camptocamp.at
+# Copyright 2015 Antiun Ingenieria, SL (Madrid, Spain)
+#        http://www.antiun.com
+#        Antonio Espinosa <antonioea@antiun.com>
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Partner Identification Numbers',
+    'category': 'Customer Relationship Management',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'contacts',
+    ],
+    'data': [
```

Comment:
IMO data sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
Done